marklogic-client-api/src/main/java/com/marklogic/client/impl/OkHttpServices.java

```diff
@@ -6270,4 +6292,28 @@ protected void set() {
         is = true;
     }
   }
+  
```

Comment:
Given that ConnectionResult most be visible to the application, should ConnectionResult be a nested static public interface of DatabaseClient with getters for the properties?  

Then, the existing ConnectionResult could be renamed as ConnectionResultImpl, implement the ConnectionResult interface, annotate the getters with @override, and change the visibility on setters to private.